### PR TITLE
Let printError be able to print bold also

### PR DESCRIPTION
**Pull request by xster**

### CODE REVIEW

File: packages/flutter_tools/lib/src/base/logger.dart [hunk 1]
```diff
@@ -25,7 +25,7 @@ abstract class Logger {
 
   /// Display an error level message to the user. Commands should use this if they
   /// fail in some way.
-  void printError(String message, [StackTrace stackTrace]);
+  void printError(String message, { StackTrace stackTrace, bool emphasis: false });
```
Comment (Hixie) [inline packages/flutter_tools/lib/src/base/logger.dart]: you give the default here but the three subclasses don't have the default

File: packages/flutter_tools/lib/src/base/logger.dart [hunk 2]
```diff
@@ -144,7 +146,10 @@ class BufferLogger extends Logger {
   String get traceText => _trace.toString();
 
   @override
-  void printError(String message, [StackTrace stackTrace]) => _error.writeln(message);
+  void printError(String message, { StackTrace stackTrace, bool emphasis }) {
+    return _error.writeln(message);
```
Comment (tvolkert) [inline packages/flutter_tools/lib/src/base/logger.dart]: No need for return

File: packages/flutter_tools/lib/src/base/logger.dart (hunk 1 above)
Comment (tvolkert) [inline packages/flutter_tools/lib/src/base/logger.dart]: Yep - in `daemon.dart` too.

**Review by tvolkert — CHANGES_REQUESTED** [review]

File: packages/flutter_tools/lib/src/base/logger.dart (hunk 2 above)
Comment (xster) [inline packages/flutter_tools/lib/src/base/logger.dart]: Done

File: packages/flutter_tools/lib/src/base/logger.dart (hunk 1 above)
Comment (xster) [inline packages/flutter_tools/lib/src/base/logger.dart]: Ah, I wasn't too sure what defaults on an abstract method's parameters actually does XD

Done

**Comment by tvolkert** [comment]
LGTM

**Review by tvolkert — APPROVED** [review]

**Pull request merged**